ci/test

```diff
@@ -199,10 +199,10 @@ _get_azure_env_args() {
   local azure_env_args=''
   azure_env_args="$azure_env_args -e AZURE_TENANT_ID=$AZURE_TENANT_ID"
   azure_env_args="$azure_env_args -e AZURE_SUBSCRIPTION_ID=$AZURE_SUBSCRIPTION_ID"
-  azure_env_args="$azure_env_args -e AZURE_RESOURCE_GROUP=$RESOURCE_GROUP"
-  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_IP=$AZURE_MACHINE_IP"
-  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_USERNAME=$AZURE_MACHINE_USERNAME"
-  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_PASSWORD=$AZURE_MACHINE_PASSWORD"
+  azure_env_args="$azure_env_args -e AZURE_RESOURCE_GROUP=$AZURE_RESOURCE_GROUP"
+  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_IP=$AZURE_AUTHN_INSTANCE_IP"
+  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_USERNAME=$AZURE_AUTHN_INSTANCE_USERNAME"
+  azure_env_args="$azure_env_args -e AZURE_AUTHN_INSTANCE_PASSWORD=$AZURE_AUTHN_INSTANCE_PASSWORD"
   azure_env_args="$azure_env_args -e USER_ASSIGNED_IDENTITY=$USER_ASSIGNED_IDENTITY"
 
 # TODO: add this once available
```

Comment:
We want to wait until you add systen-assigned-identity? b/c you have some TODOs around this

Reply:
No. The PR is big as it is. Let's merge this and add the rest of the tests later.